packages/browser/src/index.ts

```diff
@@ -29,21 +33,59 @@ export type AccessToken = {
   expiresAt: number; // Unix Timestamp in seconds
 };
 
+export const LogtoSignInSessionItemSchema = type({
+  redirectUri: string(),
+  codeVerifier: string(),
+  state: string(),
+});
+
+export type LogtoSignInSessionItem = Infer<typeof LogtoSignInSessionItemSchema>;
+
 export default class LogtoClient {
   protected accessTokenMap = new Map<string, AccessToken>();
   protected refreshToken?: string;
   protected idToken?: string;
   protected logtoConfig: LogtoConfig;
   protected oidcConfig?: OidcConfigResponse;
+  protected logtoSignInSessionKey: string;
 
   constructor(logtoConfig: LogtoConfig) {
     this.logtoConfig = logtoConfig;
+    this.logtoSignInSessionKey = getLogtoKey(logtoConfig.clientId);
   }
 
   public get isAuthenticated() {
     return Boolean(this.idToken);
   }
 
+  protected get signInSession(): Optional<LogtoSignInSessionItem> {
+    const jsonItem = sessionStorage.getItem(this.logtoSignInSessionKey);
+
+    if (!jsonItem) {
+      return undefined;
```

Comment:
```suggestion
      return;
```

Reply:
eslint does not allow.
<img width="864" alt="image" src="https://user-images.githubusercontent.com/10594507/154032468-3dda01c3-9de4-432b-a903-b7edd94245f1.png">